### DISCUSSION TITLE
Web UI: How to filter fields to show only empty rows

### DISCUSSION THREAD

**Discussion by ranmocy** [supabase/supabase#19150] [Questions]
I have a field with type non-null String, I'm trying to list out all rows that this field is empty string.
I tried many different ways but could do so.
`title is [empty input]` or `title = [empty input]` return all rows
`title is ''` or `title = ''` return no row

What's the equivalent way to filter like `where title = ''`?

<img width="346" alt="image" src="https://github.com/supabase/supabase/assets/601767/8e6e6da0-9e5f-41ae-95d2-f2770034181d">

**Comment by GaryAustin1** [comment]
I can't figure out an official way either...
But if you need something right now...   Do < filter with a single keyboard space...

**Reply by ranmocy** [reply]
That's a good tricky. Notice that it doesn't work well with unicodes, so my workaround is that, sort by the field, find the first row that is not empty, use its first char instead of the space. That works for some one-time things.

**Comment by softmarshmallow** [comment]
Not a solution within the studio, but these are one of the reasons I built Grida - A supabase admin panel,

https://github.com/gridaco/grida/pull/202

It has built-in easy filters like `Is empty` `starts with`

https://github.com/user-attachments/assets/c5a72423-154d-4d23-98be-7ebeeca1ce48

**Comment by k4chalov** [comment]
You can select `is` assignment operator. It supports null, not null, true and false.
<img width="523" height="469" alt="image" src="https://github.com/user-attachments/assets/ab7ed1ed-79b8-4f82-982b-572577c495f3" />

Not sure how long it has been added to Supabase Studio Web UI.
